docs-site/content/guide/high-availability.md

```diff
@@ -272,3 +272,137 @@ final config = Configuration(
 
   </template>
 </Tabs>
+ 
+ ## Deploy a Typesense cluster to Docker Swarm  
+
+To Deploy a Typesense cluster to Docker Swarm, follow these steps:
```

Comment:
Could we link to Docker Swarm docs here?

Reply:
Official Docker Swarm documentation?  Sure.

Reply:
will create as a sep markdown doc. i was thinking of same before